src/devtools/client/inspector/computed/components/ComputedToolbar.tsx

```diff
@@ -31,24 +25,15 @@ function ComputedToolbar(props: PropsFromRedux) {
         <input
           id="computed-searchbox"
           className="devtools-filterinput"
-          type="search"
+          type="input"
+          autoComplete="off"
           placeholder="Filter Styles"
           ref={searchFieldRef}
           onInput={setSearch}
         />
-        <button
-          id="computed-searchinput-clear"
-          className="devtools-searchinput-clear"
-          onClick={clearSearch}
-        ></button>
       </div>
       <div className="devtools-separator"></div>
-      <input
-        id="browser-style-checkbox"
-        type="checkbox"
-        className="includebrowserstyles"
-        onInput={setShowAll}
-      />
+      <Checkbox id="browser-style-checkbox" onChange={setShowAll} />
```

Comment:
Looks like the styles are not being applied. Maybe i'm on an old preview branch?


<img width="212" alt="Screen Shot 2021-12-08 at 10 17 17 AM" src="https://user-images.githubusercontent.com/254562/145262199-3a4dd4ff-4268-405c-942d-a17cc0d3b281.png">